examples/3dsphere/baroclinic_wave.jl

```diff
@@ -314,6 +314,9 @@ elseif test_name == "balanced_flow"
 end
 dt = 5
 prob = ODEProblem(rhs!, Y, (0.0, time_end))
+
+haskey(ENV, "PERF_SKIP_RUN") && exit() # for performance analysis
```

Comment:
could we use `CI_PERF_SKIP`?   all others env switches start with `CI`

Reply:
Sure, np